src/storybook/basic.stories.tsx

```diff
@@ -1,21 +1,20 @@
 import React from 'react'
 import 'photoswipe/dist/photoswipe.css'
-import { Meta, Story } from '@storybook/react'
-import { Gallery, Item } from '..'
+import { Meta, StoryObj } from '@storybook/react'
+import { Item } from '..'
+import { smallItemStyles } from './helpers/small-item-styles'
+import { withGalleryDecorator } from './helpers/with-gallery-decorator'
 
 const storyMeta: Meta = {
   title: 'Demo/Basic',
+  decorators: [withGalleryDecorator()],
 }
 
-export const basic: Story = () => {
-  const smallItemStyles: React.CSSProperties = {
-    cursor: 'pointer',
-    objectFit: 'cover',
-    width: '100%',
-    maxHeight: '100%',
-  }
-  return (
-    <Gallery>
+export default storyMeta
+
+export const Basic: StoryObj<typeof storyMeta> = {
+  render: () => {
+    return (
       <div
```

Comment:
For example, here, user will not see `<Gallery />` and it can be a kind of magic for him, why this example works in our storybook, but don't work in his code.